WordPress/gutenberg
# [Mobile] - Autocomplete - Fix regression related to the Enter key code

**Related PRs:**

- `Gutenberg Mobile` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5280

## What?
This PR fixes a regression introduced by https://github.com/WordPress/gutenberg/pull/45510 where it changed getting the keyboard code from the `key` attribute which is being deprecated in the Web standards to `code`.

This is related to a previous effort (see https://github.com/WordPress/gutenberg/pull/43521) to update the mobile code to this new attribute.

## Why?
To fix the functionality of the autocomplete also known as slash inserter.

## How?
Now it will add the `code` attribute through the `customEditableOnKeyDown` callback event.

## Testing Instructions
- Open the editor
- In a Paragraph block type in `/image` and tap on the return key of the keyboard
- **Expect** to see the Image block created.

## Screenshots or screencast 
<kbd><img src="https://user-images.githubusercontent.com/4885740/203538408-5efab0cc-0b11-4e8c-835f-135daa9a28f3.gif" width=250 /></kbd>
